db/migrate/20220929204746_add_content_view_environment_content_facet.rb

```diff
@@ -37,16 +38,47 @@ def down
 
     ::Katello::Host::ContentFacet.reset_column_information
 
+    default_cv_id = ::Katello::ContentView.default.first.id
```

Comment:
Are these changes necessary.. I ;d think down should just fail with not implemented error.?

Reply:
Down is needed for devs who don't want to torture themselves :) it doesn't make much difference for users or customers

I did have to make these changes so that my down migration would work.